build.gradle

```diff
@@ -68,7 +68,6 @@ allprojects {
         compile group: 'org.facebook4j', name: 'facebook4j-core', version: '2.4.10'
         compile group: 'com.restfb', name: 'restfb', version: '1.6.9'
         compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '2.41.0'
```

Comment:
can we remove selenium as well?

Reply:
Or is this used for our tests?

Reply:
Ah I will see if I can remove it next week, not too sure where is affects yet

Reply:
ok i just tried, removed it!